packages/contracts/hardhat.config.ts

```diff
@@ -158,6 +158,7 @@ const config: HardhatUserConfig = {
   solidity: {
     version: '0.8.17',
     settings: {
+      viaIR: true,
```

Comment:
the `solidity-coverage` that the project is using has problems with `viaIR: true` I have to update to lates version to be able to run `hh coverage`